subprojects/GOPATH/src/github.com/microsoft/hivedscheduler/pkg/api/types.go

```diff
@@ -158,3 +158,32 @@ type LazyPreemptionStatus struct {
 	// It was lazy preempted at PreemptionTime.
 	PreemptionTime meta.Time `json:"preemptionTime"`
 }
+
+type VirtualClusterList struct {
+	Items []VirtualCluster `json:"items"`
+}
+
+type VirtualCluster struct {
+	ObjectMeta `json:"metadata"`
+	Status     VirtualClusterStatus `json:"status"`
+}
+
+type VirtualClusterStatus struct {
+	VirtualClusterCapacity *ClusterCapacity `json:"virtualClusterCapacity"`
+}
+
+type ClusterCapacity struct {
+	Total                      int32 `json:"total"`
+	GuaranteedUsage            int32 `json:"guaranteedUsage"`
```

Comment:
>GuaranteedUsage [](start = 1, length = 15)

Capacity generally means Total in K8s context, so just use VirtualClusterStatus?

Such as
type VirtualClusterStatus struct {
UsedGpu
TotalGpu
BestCaseFreeGpu
WorstCaseFreeGpu
OpportunisticUsedGpu // If no Opportunistic prefix, all fields are about GT.
}